src/main/java/io/anserini/document/CarDocument.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Anserini: An information retrieval toolkit built on Lucene
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.document;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/**
+ * A paragraph object in the CAR dataset ver2.0. The paraID serves as the id.
+ * Reference: http://trec-car.cs.unh.edu/datareleases/
+ */
+public class CarDocument implements SourceDocument {
+  private final String paraID;
+  private final String paragraph;
+
+  public CarDocument(String paraID, String paragraph) {
+    this.paraID = paraID;
+    this.paragraph = paragraph;
+  }
+
+  /**
```

Comment:
The javadoc here does not tell anything.
Instead, you can explain here why this function always returns `null` which other document adaptors usually do not, e.g. because the documents are processed somewhere else.

Reply:
Good suggestion.  Thanks.